docs/docs/installation/environment-set-up.mdx

```diff
@@ -16,9 +16,6 @@ pip3 --version
 :::note Python3 Supported Versions
 
 Currently, rasa supports the following Python versions: `3.7`, `3.8`, `3.9` and `3.10`.
-Note that Python `3.10` is not currently supported for rasa installations on Apple Silicon.
-The blocker consists of current inability to update to Tensorflow version >= `2.10.x` which is pending the resolution
-of a Tensorflow bug.
```

Comment:
We need to specify here that  Python `3.10` on Apple Silicon doesn't work in `3.4.x`, but will be supported from `3.5.x` onwards.